### DISCUSSION TITLE
Default Prompt Suggestions never get removed

### DISCUSSION THREAD

**Discussion by eramax** [open-webui/open-webui#906] [General]
when i delete all the Default Prompt Suggestions, it appears again.

**Comment by rudism** [comment]
I was having the same issue, and discovered that you can configure these in a persistent way by copying [config.json](https://github.com/open-webui/open-webui/blob/main/backend/data/config.json) to your data directory and setting it to an empty array:

```json
{
  "version": 0,
  "ui": {
    "default_locale": "en-US",
    "prompt_suggestions": []
  }
}
```

Making changes through the UI doesn't seem to modify that file, so it'll reset to whatever is in there each time you start up.

**Reply by imbible** [reply]
This is not working for me. The default prompt suggestions are hardcoded in python and overwrite my config every time I restart the docker.
https://github.com/open-webui/open-webui/blob/a8d80f936a69c7e4254c2306b9c88aa7c3e09088/backend/config.py#L514
The `"prompt_suggestions": []` remain empty in config.json, but the default 6 suggestions just show up in UI.

**Reply by devbysb** [reply]
Same problem have you found a solution ?

**Reply by silentoplayz** [reply]
@imbible, @devbysb [To disable `PersistentConfig` and have Open WebUI treat all variables equally, you can set `ENABLE_PERSISTENT_CONFIG` to `False`.](https://docs.openwebui.com/getting-started/env-configuration) I believe this may be your solution.

`DEFAULT_PROMPT_SUGGESTIONS` [documentation](https://docs.openwebui.com/getting-started/env-configuration#default_prompt_suggestions) doesn't mention that it is a `PersistentConfig` environment variable, which is a small mistake.
